training/genetic/models.py

```diff
@@ -0,0 +1,47 @@
+from django.db import models
+from django.core.validators import MaxValueValidator, MinValueValidator
+
+
+# Create your models here.
+
+class OptimumDNA(models.Model):
+    DNA = models.CharField(max_length=100)
+    #pub_date = models.DateTimeField('date published')
+
+    def __str__(self):
+        return self.DNA
+
+class Individual(models.Model):
+
+    optimum_DNA = models.ForeignKey(OptimumDNA, on_delete=models.CASCADE)
+    DNA = models.CharField(max_length=100)
```

Comment:
Why do you have a DNA here and in OptimumDNA, if they will be the same DNAS this one is redundant.

Reply:
In the population, there are Individuals. OptimumDNA is the ultimate DNA for the population, it is not an Individual. 

*OptimumDNA* is a DNA that Individuals may relate to.
*DNA* is DNA of the Individual, there can be many Individuals.

For example:
OptimumDNA : 
 - "Hello world"
Individuals : 
 - "Hello corld"
 - "Yello world"
 - "Yello corld"